docs/extensions/custom-auth/_category_.json

```
@@ -0,0 +1,5 @@
+{
+    "label": "Signature Authenticators",
```

```suggestion
    "label": "Authenticators",
```

aligns with title in extensions overview

---

The other labels do not align with the extensions overview.